internal/util/sort.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+
+// BundlesByCreationTimestamp sorts a list of ReplicaSet by creation timestamp, using their names as a tie breaker.
+type BundlesByCreationTimestamp []*rukpakv1alpha1.Bundle
+
+func (o BundlesByCreationTimestamp) Len() int      { return len(o) }
+func (o BundlesByCreationTimestamp) Swap(i, j int) { o[i], o[j] = o[j], o[i] }
+func (o BundlesByCreationTimestamp) Less(i, j int) bool {
+	if o[i].CreationTimestamp.Equal(&o[j].CreationTimestamp) {
+		return o[i].Name < o[j].Name
+	}
+	return o[i].CreationTimestamp.Before(&o[j].CreationTimestamp)
+}
```

Comment:
Any consideration for using `sort.Slice` with this `Less` function re-implemented as an anonymous function?

Reply:
That seems like an interesting implementation. I think I can simplify the sorting mechanism anyways so I'll play around with that locally and push up a separate commit if I can something working.

Reply:
Updated the implementation to use sort.Slice in https://github.com/operator-framework/rukpak/pull/293/commits/bd136663720eb0439aef70265af7b29c88d4e586